### DISCUSSION TITLE
Make "prevew mode" default, and allow unlimited custom color presets, instead of two.

### DISCUSSION THREAD

**Discussion by legalizecannabis** [darkreader/darkreader#9786] [Ideas]
Make "preview mode" the default for all darkreader installations.  It only makes sense, but I'd ask that you make it obvious and not hidden in "dev tools" that no-one will ever even think of moving the mouse towards, to access the color options.

I also want unlimited amount of themes, or "light / dark" options.  Two is not nearly enough once given the ability to pick any brightness and color.

**Comment by Gusted** [comment]
> Make "preview mode" the default for all darkreader installations. It only makes sense, but I'd ask that you make it obvious and not hidden in "dev tools" that no-one will ever even think of moving the mouse towards, to access the color options.


I've already been in favor of making the preview mode the default mode (and yank out the old UI). However it's currently considered (not by me) that's "unpolished and not complete" for as long as its existence. I'm still waiting on having some kind of update for this new UI that is considered complete and polished, but I might just turn the switch soon. Quite a few incoming issues every week are already implemented/solved if you would have the new UI, it's just sad that this old design is still being used.

**Reply by legalizecannabis** [reply]
I'm sure there are unknown bugs to work on, but have you any plans to add addition color scheme slots?  I'm sure I could eventually figure the code out, just looking at "Light" and "Dark" and just copy that, and rename it something else, and add a + plus button to add that interactively, as many as desired.

**Reply by KaKi87** [reply]
Hi,

I came here to ask the same thing as OP's first ask : please make the "preview" default.

I've been using it for ages too, and get surprised every time I install a new browser or a new computer and notice I still have to do all those additional clicks to get something I consider to have been stable for years.

Thank you
